interface/billing/sl_eob_search.php

```diff
@@ -1,849 +1,832 @@
-      "p.fname, p.mname, p.lname, p.street, p.city, p.state, p.postal_code, p.billing_note as pat_billing_note " .
-      "FROM form_encounter AS f, patient_data AS p " .
-      "WHERE ( $where ) AND " .
-      "p.pid = f.pid " .
-      "ORDER BY p.lname, p.fname, f.pid, f.date, f.encounter", $sqlBindArray);
-
-    $stmt = array();
-    $stmt_count = 0;
-
-    // This loops once for each invoice/encounter.
-    //
-    while ($row = sqlFetchArray($res)) {
-      $svcdate = substr($row['date'], 0, 10);
-      $duedate = $svcdate; // TBD?
-      $duncount = $row['stmt_count'];
-      $enc_note = $row['enc_billing_note'];
-
-      // If this is a new patient then print the pending statement
-      // and start a new one.  This is an associative array:
-      //
-      //  cid     = same as pid
-      //  pid     = LibreHealth EHR patient ID
-      //  patient = patient name
-      //  amount  = total amount due
-      //  adjust  = adjustments (already applied to amount)
-      //  duedate = due date of the oldest included invoice
-      //  age     = number of days from duedate to today
-      //  to      = array of addressee name/address lines
-      //  lines   = array of:
-      //    dos     = date of service "yyyy-mm-dd"
-      //    desc    = description
-      //    amount  = charge less adjustments
-      //    paid    = amount paid
-      //    notice  = 1 for first notice, 2 for second, etc.
-      //    detail  = array of details, see invoice_summary.inc.php
-      //
-      if ($stmt['cid'] != $row['pid']) {
-        fwrite($fhprint, make_statement($stmt));
-        if (!empty($stmt)) ++$stmt_count;
-        $stmt['cid'] = $row['pid'];
-        $stmt['pid'] = $row['pid'];
-        $stmt['dun_count'] = $row['stmt_count'];
-        $stmt['bill_note'] = $row['pat_billing_note'];
-        $stmt['enc_bill_note'] = $row['enc_billing_note'];
-        $stmt['bill_level'] = $row['last_level_billed'];
-        $stmt['level_closed'] = $row['last_level_closed'];
-        $stmt['patient'] = $row['fname'] . ' ' . $row['lname'];
-      $stmt['encounter'] = $row['encounter'];
-        #If you use the field in demographics layout called
-        #guardiansname this will allow you to send statements to the parent
-        #of a child or a guardian etc
-      if(strlen($row['guardiansname']) == 0) {
-        $stmt['to'] = array($row['fname'] . ' ' . $row['lname']);
-        } else {
-       $stmt['to'] = array($row['guardiansname']);
-     }
-        if ($row['street']) $stmt['to'][] = $row['street'];
-        $stmt['to'][] = $row['city'] . ", " . $row['state'] . " " . $row['postal_code'];
-        $stmt['lines'] = array();
-        $stmt['amount'] = '0.00';
-        $stmt['ins_paid'] = 0;
-        $stmt['today'] = $today;
-        $stmt['duedate'] = $duedate;
-      } else {
-        // Report the oldest due date.
-        if ($duedate < $stmt['duedate']) {
-          $stmt['duedate'] = $duedate;
+  
+  function bucks($amount) {
```

Comment:
Geez...  function bucks is defined all over the place.  I think there is a better way...one day we will get to that.